doc/vxlan/EVPN/EVPN_VXLAN_HLD.md

```diff
@@ -0,0 +1,1023 @@
+
+key = EVPN_NVO_TABLE:nvo_name
+                          ; nvo or VTEP name as a string
+; field = value
+source_vtep = tunnel_name ; refers to the tunnel object created with SIP only.
+```
+
+**NEIGH_SUPPRESS_CFG_VLAN_TABLE**
+
+Producer:  config manager
+
+Consumer: VlanMgr
+
+Description: New table that stores neighbor suppression per VLAN configuration.
+
+Schema:
+
+```
+;New table
+;Stores Neigh Suppress configuration per VLAN
+
+key             = NEIGH_SUPPRESS_VLAN:"Vlan"vlanid       ; VLAN with prefix "NEIGH_SUPPRESS"
+suppress_neigh  = "ON" / "OFF" ; Default "OFF"
+
+```
+
+**VRF_TABLE**
+
+Producer:  config manager
+
+Consumer: VrfMgr
+
+Description: Updated existing table to store VNI associated with VRF.
+
+Schema:
+
+```
+;Existing table
+;defines virtual routing forward table. Updated to stores VNI associated with VRF
+;
+;Status: stable
+
+key = VRF_TABLE:VRF_NAME ;
+fallback = "true"/"false"
+vni = 1*8DIGIT ; VNI associated with VRF
+
+```
+
+### 4.2.2 APP_DB Changes
+
+**EVPN_NVO_TABLE**
+
+Producer:  VxlanMgr 
+
+Consumer: VxlanOrch
+
+Description: This is a singleton configuration which holds the VTEP source IP used for BGP-EVPN discovered tunnels. 
+
+Schema:
+
+```
+; New table
+; holds the VTEP source IP used for BGP-EVPN discovered tunnels
+
+key = EVPN_NVO_TABLE:nvo_name
+                          ; nvo or VTEP name as a string
+; field = value
+source_vtep = tunnel_name ; refers to the VXLAN_TUNNEL object created with SIP only.
+```
+
+**EVPN_REMOTE_VNI** 
+
+Producer: fdbsyncd
+
+Consumer: VxlanOrch
+
+Description: This specifies the remote VNI to be extended over a tunnel. This is populated by fdbsyncd on receipt of a IMET route from the remote side. 
+
+Schema:
+
+```
+; New table
+; specifies the VLAN to be extended over a tunnel
+
+key = EVPN_REMOTE_VNI_TABLE:"Vlan"vlanid:remote_vtep_ip
+                  ; Vlan ID and Remote VTEP IP 
+; field = value
+vni = 1*8DIGIT    ; vni to be used for this VLAN when sending traffic to the remote VTEP
+```
+
+**VXLAN_FDB_TABLE**
+
+Producer: fdbsyncd
+
+Consumer: fdborch
+
+Description: This holds the MACs learnt over a VNI from the remote VTEP.  This is populated by fdbsyncd on receipt of a MAC route. 
+
+Schema:
+
+```
+; New table
+; specifies the MAC and VLAN to be programmed over a tunnel
+
+key = VXLAN_FDB_TABLE:"Vlan"vlanid:mac_address
+                          ;MAC Address and VLAN ID
+;field = value
+remote_vtep = ipv4
+type          = "static" / "dynamic"  ; type of the entry.
+vni         = 1*8DIGIT                ; vni to be used for this VLAN when sending traffic to the remote VTEP
+```
+
+**VRF_TABLE**
+
+Producer: VrfMgr
+
+Consumer: Vrforch
+
+Description: Updated existing table to store VNI associated with VRF.
+
+Schema:
+
+```
+;Existing table
+;defines virtual routing forward table. Updated to stores VNI associated with VRF
+;
+;Status: stable
+
+key = VRF_TABLE:VRF_NAME ;
+fallback = "true"/"false"             ; default false
```

Comment:
What is intended behavior with fallback option?

Reply:
This is existing field in VRF Table. Please refer Following VRF HLD for detail.
https://github.com/Azure/SONiC/blob/master/doc/vrf/sonic-vrf-hld.md